src/core/server.ts

```diff
@@ -99,6 +101,10 @@ export class Server<C extends BaseContext> {
         : process.env.NODE_ENV === 'development';
 
     this.resolversPath = this.appOptions.resolversPath || [process.cwd() + '/**/*.resolver.ts'];
+
+    if (typeof this.appOptions.introspectionEnabled !== 'undefined') {
+      this.introspectionEnabled = this.appOptions.introspectionEnabled;
+    }
```

Comment:
Since this is only being used once, we could just do `!!this.appOptions.introspection` below.

Reply:
Ok, didn't know about !!